File: December/Javascript/Day5.js
// Selection sort 

function  OrderedArray(arr){

    for(let i = 0 ; i <= arr.length - 1 ; i++){
        minIdx = i 

        for(let j = 0 ; j < arr.length ; j++  ){
            if(arr[j] > arr[minIdx]){
                minIdx = j 
            }
            // else return

            if(minIdx != i){
                let temp = arr[i]
                arr[i] = arr[minIdx]
                arr[minIdx] = temp
             }
        }
        
        
        
    }
    console.log(arr);

}OrderedArray( [13, 89, 15, 28, 1, 7])

// let a = 5
// console.log(a);


// let sum = 0 ;
// let i = 1
// while(i <= 5){
//     sum += i
//     i++
// }
// console.log(sum);

File: December/Python/Day5.py
#Implementing  Selection Sort Optimized code for selection sort
arr = [13, 89, 15, 28, 1, 7]

for i in range(0, len(arr) - 1): # range doesn't include the last number
    minIdx = i
    #finding the minimum index in the arr and swapping it
    for j in range(i + 1, len(arr)):
        if(arr[j] < arr[minIdx]):
            minIdx = j

        else:
            continue
    
    #Swapping value in ith index with the min index
    if(minIdx != i):
        arr[i], arr[minIdx] = arr[minIdx], arr[i]
    else:
        break

print(arr)

#Numeric expression
xx = 2
xx = xx + 2
print(xx)

#
x = 2
x = (3*x) - 1
print(x)